Models/CertificateValidationRequest.swift

```diff
@@ -91,11 +95,13 @@ class CertificateValidationRequest {
     }
     
     private func computeLocalHash() {
-        localHash = sha256(data: certificate.file)
+        self.localHash = sha256(data: certificate.file)
```

Comment:
fun fact! `self.` is implicit unless you've declared a new variable in the same scope with an identical name. 

/play tmyk